From b509c886165f0d9ae475e29d7b0e5945a52d0343 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Fri, 19 May 2017 11:48:04 +0900
Subject: clk: renesas: Add r8a77995 CPG Core Clock Definitions

This patch adds macros usable by the device tree sources to reference
the R8A77995 CPG clocks by index. The data comes from Table 8.2f in
DRAFT of the R-Car D3 CPG  User's Manual.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
---
 include/dt-bindings/clock/r8a77995-cpg-mssr.h | 58 +++++++++++++++++++++++++++
 1 file changed, 58 insertions(+)
 create mode 100644 include/dt-bindings/clock/r8a77995-cpg-mssr.h

diff --git a/include/dt-bindings/clock/r8a77995-cpg-mssr.h b/include/dt-bindings/clock/r8a77995-cpg-mssr.h
new file mode 100644
index 0000000..1b827b4
--- /dev/null
+++ b/include/dt-bindings/clock/r8a77995-cpg-mssr.h
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2017 Renesas Electronics Corp.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ */
+#ifndef __DT_BINDINGS_CLOCK_R8A77995_CPG_MSSR_H__
+#define __DT_BINDINGS_CLOCK_R8A77995_CPG_MSSR_H__
+
+#include <dt-bindings/clock/renesas-cpg-mssr.h>
+
+/* r8a77995 CPG Core Clocks */
+#define R8A77995_CLK_Z2			1
+#define R8A77995_CLK_ZG			2
+#define R8A77995_CLK_ZTR		3
+#define R8A77995_CLK_ZT			4
+#define R8A77995_CLK_ZX			5
+#define R8A77995_CLK_USB		6
+#define R8A77995_CLK_S0D1		7
+#define R8A77995_CLK_S1D1		8
+#define R8A77995_CLK_S1D2		9
+#define R8A77995_CLK_S1D4		10
+#define R8A77995_CLK_S2D1		11
+#define R8A77995_CLK_S2D2		12
+#define R8A77995_CLK_S2D4		13
+#define R8A77995_CLK_S3D1		14
+#define R8A77995_CLK_S3D2		15
+#define R8A77995_CLK_S3D4		16
+#define R8A77995_CLK_S1D4C		17
+#define R8A77995_CLK_S3D1C		18
+#define R8A77995_CLK_S3D2C		19
+#define R8A77995_CLK_S3D4C		20
+#define R8A77995_CLK_LB			21
+#define R8A77995_CLK_CL			22
+#define R8A77995_CLK_ZB3		23
+#define R8A77995_CLK_ZB3D2		24
+#define R8A77995_CLK_CR			25
+#define R8A77995_CLK_CRD2		26
+#define R8A77995_CLK_SD0H		27
+#define R8A77995_CLK_SD0		28
+#define R8A77995_CLK_SSP2		29
+#define R8A77995_CLK_SSP1		30
+#define R8A77995_CLK_RPC		31
+#define R8A77995_CLK_RPCD2		32
+#define R8A77995_CLK_ZA2		33
+#define R8A77995_CLK_ZA8		34
+#define R8A77995_CLK_Z2D		35
+#define R8A77995_CLK_CANFD		36
+#define R8A77995_CLK_MSO		37
+#define R8A77995_CLK_RCK		38
+#define R8A77995_CLK_OSC		39
+#define R8A77995_CLK_LV0		40
+#define R8A77995_CLK_LV1		41
+#define R8A77995_CLK_CP			42
+
+#endif /* __DT_BINDINGS_CLOCK_R8A77995_CPG_MSSR_H__ */
-- 
1.9.1

